tools/downloadSyncClient.sh

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env bash
```

Comment:
I actually think the new normal is to name things tools/downloadSyncClient not tools/downloadSyncClient.sh unless you really are a .sh script (not bash)